zulipterminal/config/tutorial.py

```diff
@@ -0,0 +1,24 @@
+TUTORIAL = """
+
+Basic workflows
+
+A few basic workflows are useful when getting oriented.
+
+Moving around
+
+- You can use arrow keys or `j`/`k`/`l`/`m` keys to move around and
+ `esc` to go back or move out of narrow.
+
+For reading
+
+- Use the left sidebar (or the `n` key) to review messages thread-by-thread.
```

Comment:
Maybe "review unread messages topic-by-topic", instead? Also, we can navigate through unread streams using the left sidebar, but unread topics using `n`, so maybe separate those two?